src/main/java/org/zalando/nakadi/domain/SubscriptionCursor.java

```diff
@@ -40,8 +43,8 @@ public boolean equals(final Object o) {
     @Override
     public int hashCode() {
         int result = super.hashCode();
-        result = 31 * result + eventType.hashCode();
-        result = 31 * result + cursorToken.hashCode();
+        result = 31 * result + (eventType != null ? eventType.hashCode() : 0);
```

Comment:
@v-stepanov understood, I think it is better to have equals and hashcode methods consistent.